care/facility/models/patient.py

```diff
@@ -1,5 +1,7 @@
+import datetime
+
 from django.db import models
-from fernet_fields import EncryptedCharField, EncryptedTextField
+from fernet_fields import EncryptedCharField, EncryptedIntegerField, EncryptedTextField
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'fernet_fields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5332931)